src/tests/ov_helpers/ov_snippets_models/src/fake_quantize_helper.cpp

```diff
@@ -189,6 +189,56 @@ std::shared_ptr<ov::Model> FakeQuantizeFunction::getSubgraphWithFakeQuantize(
     return function;
 }
 
+std::shared_ptr<ov::Node> FakeQuantizeFunction::getDecomposedFakeQuantizeOps(const ov::Output<ov::Node>& input,
+                                                                             const ov::element::Type outType,
+                                                                             bool doRounding,
+                                                                             bool doDequantize) {
+    auto inputShape = input.get_shape();
+    auto inputType = input.get_element_type();
+
+    const auto input_low = ov::op::v0::Constant::create(ov::element::f32, {1}, {1.f});
+    const auto input_high = ov::op::v0::Constant::create(ov::element::f32, {1}, {20.f});
+    const auto output_scale = ov::op::v0::Constant::create(ov::element::f32, {1}, {13.4211f});
```

Comment:
I think we need to pass these values as helper parameters. We have at least 2 common scenarios: signed and unsigned interval

Reply:
Should we make any default values?